core/src/banking_stage/forwarder.rs

```diff
@@ -228,11 +228,13 @@ impl Forwarder {
         usize,
         Option<Pubkey>,
     ) {
-        let (res, num_packets, forward_us, leader_pubkey) =
+        let (res, num_packets, _forward_us, leader_pubkey) =
             self.forward_packets(forward_option, forwardable_packets);
+        if let Err(ref err) = res {
+            warn!("failed to forward packets: {err}");
+        }
 
         if num_packets > 0 {
-            inc_new_counter_info!("banking_stage-forwarded_packets", num_packets);
```

Comment:
This covered by `banking_stage_stats.forwarded_vote_count` and `banking_stage.forwarded_transaction_count`